lms/djangoapps/branding/api.py

```diff
@@ -329,3 +330,63 @@ def _absolute_url_staticfile(is_secure, name):
     # For local development, the returned URL will be relative,
     # so we need to make it absolute.
     return _absolute_url(is_secure, url_path)
+
+
+def get_microsite_url(name):
+    """
+    Look up and return the value for given url name in microsite configuration.
+    Return '#' if given url name is not defined in microsite configuration.
+    """
+    url = microsite.get_value(name)
+    if url:
+        return url
+    else:
+        return EMPTY_URL
+
+
+def get_url(name):
+    """
+    Lookup and return page url, lookup is performed in the following order
+
+    1. get marketing link, if marketing is off then platform url will be used
```

Comment:
If you change the implementation as suggested below, you will also need to update the docstring